yarn.lock

```diff
@@ -1616,13 +1616,13 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@ledgerhq/types-live@npm:^6.52.0":
-  version: 6.52.0
-  resolution: "@ledgerhq/types-live@npm:6.52.0"
+"@ledgerhq/types-live@npm:^6.52.0, @ledgerhq/types-live@npm:^6.54.0":
```

Comment:
Actually, I did not check the other ledger deps, but looks like some other dependencies were also using `6.52.0`, so we might wanna use that version too.

Or we could also update them all, but since you have done your tests with the `6.52.0` I would just downgrade it.

WDYT?

Reply:
ok, i will downgrade the version to the one we have done the testing.